SummerProject/static/src/components/styles/mainLogInFlow.css

```diff
@@ -0,0 +1,226 @@
+.login-page {
+    display: flex;
+    justify-content: flex-start;
+    max-width: 1440px;
+    padding: 0;
+    background: white;
+}
+
+.right-part {
+    align-self: center;
+    margin: auto;
+}
+
+.left-part {
+    position: relative;
+    max-width: 640px;
+    width: 100%;
+    height: 100vh;
+    overflow: hidden;
+}
+
+.logo {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    background: #D72130;
+    width: 245px;
+    height: 83px;
+    text-align: center;
+}
+
+a.logo-link {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    width: 50%;
+    height: 30%;
+    margin: auto;
+    text-decoration: none;
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 28px;
+    color: #FFFBFB;
+}
+
+.right-part p {
+    margin-bottom: 30px;
+}
+
+.log-in {
+    position: absolute;
+    top: 0;
+    right: 0;
+    display: flex;
+    width: 375px;
+    height: 83px;
+    text-align: center;
+    justify-content: center;
+    z-index: 1;
+
+}
+
+a.login-link {
+    position: absolute;
+    left: 15%;
+    top: 26%;
+    bottom: 6.67%;
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 17px;
+    text-align: center;
+    color: #596377;
+}
+
+a.login-link-mobile{
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 17px;
+    color: #FFFFFF;
+}
+
+.right-part h1 {
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 33px;
+    color: #000000;
+}
+
+.right-part p {
+    font-family: sans-serif;
+    font-style: normal;
```

Comment:
Do you ever need `font-family` other than `sans-serif` and `font-style` other than `normal `? You're duplicating these style rules pretty often. You could set these rules for some parent element and all it's child would share them